### DISCUSSION TITLE
Gap attribute is not working

### DISCUSSION THREAD

**Discussion by garmistry** [tailwindlabs/tailwindcss#10951] [Help]
**What version of Tailwind CSS are you using?**

For example: v3.3.1

**What build tool (or framework if it abstracts the build tool) are you using?**

"next": "13.2.4",

**What browser are you using?**

Chrome,

**What operating system are you using?**

MacOS

**Describe your issue**

I was attempting to use the 'gap' flag with TailwindCSS 3.3.1 and I could not see the elements inherit the attribute when creating Card objects. 

After I downgraded to 2.2.19, the gap feature work and applied the items correctly. Might be a singular issue but maybe someone can test? 

I was following the multi-card examples from the Tremor Dashboard Setups. 

https://www.tremor.so/docs/getting-started/demo-dashboard

**Comment by opheliagame** [comment]
@garmistry 
is this issue still persisting?
